cg/services/order_validation_service/utils.py

```diff
@@ -45,3 +46,7 @@ def apply_sample_validation(rules: list[Callable], order: Order, store: Store) -
 
 def parse_buffer(buffer: str | None) -> ElutionBuffer | None:
     return ElutionBuffer.OTHER if buffer and buffer.startswith("Other") else buffer
+
+
+def parse_control(control: ControlEnum | None) -> ControlEnum:
```

Comment:
Or something like that
```suggestion
def parse_control(control: ControlEnum | None) -> ControlEnum:
    """Convert the control value into one of the Enum values if is None."""
```